docs/glossary.md

```diff
@@ -17,45 +17,65 @@ AWS
-    [DigitalOcean, Inc.](https://www.digitalocean.com/) is an American cloud infrastructure provider headquartered in New York City with data centers worldwide.
-
-ZODB
-    [A native object database for Python](https://zodb.org/en/latest/).
-
 Barceloneta
     The default theme for Plone 5.
 
 CMS
     Content Management System
 
+cookiecutter
+    A command-line utility that creates projects from cookiecutters (project templates), e.g. creating a Python package project from a Python package project template.
+    [See Cookiecutter's documentation](https://cookiecutter.readthedocs.io/en/stable/).
+
+cookiecutter-zope-instance
+    [cookiecutter-zope-instance](https://github.com/plone/cookiecutter-zope-instance) is a cookiecutter template to create a full and complex configuration of a Zope WSGI instance.
+
 CSS
     Cascading Style Sheets (CSS) is a stylesheet language used for describing the (most of the times visual) representation of web pages.
 
+DigitalOcean
+    [DigitalOcean, Inc.](https://www.digitalocean.com/) is an American cloud infrastructure provider headquartered in New York City with data centers worldwide.
+
 Grunt
     The JavaScript Task Runner.
     Automates the creation and manipulation of static assets for the theme.
 
 Less
     A dynamic stylesheet language that can be compiled into {term}`CSS` (Cascading Style Sheets).
 
+Linode
+    [Linode.com](https://www.linode.com/) is an American privately owned virtual private server provider company based in Galloway, New Jersey, United States.
+
+mxdev
+    [mxdev](https://github.com/mxstack/mxdev) [mɪks dɛv] is a utility that makes it easy to work with Python projects containing lots of packages, of which you only want to develop some.
+    It builds on top of the idea to have stable version constraints and then develop from a VCS on top of it.
+    As part of above use-case sometimes versions of the stable constraints need an override with a different (i.e. newer) version.
+
+mxmake
+    mxmake helps create development environments for Python projects based on [make](https://www.gnu.org/software/make) and [mxdev](https://github.com/mxstack/mxdev).
+
+NFS
+    [Network File System](https://en.wikipedia.org/wiki/Network_File_System).
+
+
 NPM
     npm is a package manager for the JavaScript programming language.
     It is the default package manager for the JavaScript runtime environment Node.js.
     Also a registry of JavaScript packages, similar to PyPI.
 
-TTW
-    Through-The-Web, changes in the browser.
+pip
+    pip is the package installer for Python.
+    See [tool recommendations](https://packaging.python.org/en/latest/guides/tool-recommendations/) for more information.
 
 S3
     [Amazon Web Services S3](https://aws.amazon.com/s3/).
     Object storage built to store and retrieve any amount of data from anywhere.
 
-NFS
-    [Network File System](https://en.wikipedia.org/wiki/Network_File_System).
+TTW
+    Through-The-Web, changes in the browser.
+
+
+
```

Comment:
Delete extra blank lines.

```suggestion
```

Reply:
hm, can we sort the glossary alphabetically, it would peace my mind.

Reply:
Line 14 sorts when rendered.

```
:sorted: true
```

Or do you mean manually sort the entries, and require developers to sort new entries going forward? If so, we should document that requirement in our contributing docs.

Reply:
well, lets leave it by now unsorted as it is.